github.md

```diff
@@ -0,0 +1,9 @@
+# hello
+
+It is 30 degress in Hamburg.
+It is getting hotter compared to last years.
+Next year I am planing to buy an AC.
+
+My Summer Holiday Essay
```

Comment:
I  would recommend some text formatting right here. Something like ##... for better readability